heap/heap.cpp

```diff
@@ -0,0 +1,53 @@
+#include "heap.h"
+#include <vector>
+#include <iostream>
+
+void Heap::sift_down(int indx){
+    while (2*indx+1<array.size()){
+        int left=2*indx+1;
+        int right=2*indx+2;
+        int buffer=left;
+
+        if (right<array.size() && array[right]>array[left]){
```

Comment:
Почему здесь у одного бинарного оператора (<) стоят пробелы вокруг, а у другого (&&) нет? Если нет консистетнтности в установке пробелов, то выбери один из вариантов, я рекомендую отделять все бинарные операторы пробелами.
Либо скинь ссылку на code-style, которому хочешь следовать и оформи также, как там
Если не хочешь их искать, то, например, этот кусок я бы оформил так:
```suggestion
        if (right < array.size() && array[right] > array[left]) {
```

Ну и все твои другие места тоже так поправь (это всё один коммит)